### PR TITLE
make blktests to run for different disk type

### PR DESCRIPTION
**Pull request by smruti77**
add a yaml to run for disk devices and make chnaged
in script to run for different disk.

Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

**Comment by smruti77** [comment]
[blktests.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/5134544/blktests.log)
[blktests_loop.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/5134546/blktests_loop.log)

**Comment by smruti77** [comment]
JOB ID     : cf0f025ba20ef0a55f2940ec13972a26cb89914e
JOB LOG    : /root/avocado/job-results/job-2020-08-27T02.15-cf0f025/job.log
 (1/1) blktests.py:Blktests.test;run-a792: PASS (426.14 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2020-08-27T02.15-cf0f025/results.html
JOB TIME   : 426.81 s

File: fs/blktests.py [hunk 1]
```diff
@@ -59,7 +61,14 @@ def test(self):
...
         genio.write_one_line("/proc/sys/kernel/hung_task_timeout_secs", "0")
-        process.system('./check', ignore_status=True)
+        if self.disk:
+            os.environ['TEST_DEVS'] = self.disk
+            cmd = './check %s' % self.dev_type
+            result = process.run(cmd, ignore_status=True, verbose=True)
+            if result.exit_status != 0:
+                self.fail("test failed")
+        else:
+            process.system('./check', ignore_status=True)
```
Comment (narasimhan-v) [inline fs/blktests.py:71]:
```
        if self.disk:
            os.environ['TEST_DEVS'] = self.disk
        cmd = './check %s' % self.dev_type
        result = process.run(cmd, ignore_status=True, verbose=True)
        if result.exit_status != 0:
            self.fail("test failed")
```

**Review by narasimhan-v — CHANGES_REQUESTED** [review]

File: fs/blktests.py (hunk 1 above)
Comment (smruti77) [reply fs/blktests.py:71]: @narasimhan-v actually  test running before like this  './check'. so i left that in else part. if someone don't want to run using disk and type like before they can run.

**Review by narasimhan-v — APPROVED** [review]

**Pull request merged**